platforms/openstack/modules/vms/vms.tf

```diff
@@ -58,6 +58,11 @@ module "tags" {
   }
 }
 
+resource "openstack_compute_servergroup_v2" "nodes" {
+  name     = "${var.prefix}-servergroup"
+  policies = ["anti-affinity"]
+}
+
```

Comment:
This way three server groups with the SAME name are generated (one for bastion, worker and master).
It should be moved to the iaas module

Reply:
Should be fixed now.

Reply:
Moved the server group into iaas module.